src/20251113_AaveV3Ethereum_GSMMigration/AaveV3Ethereum_GSMMigration_20251113.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20} from 'openzeppelin-contracts/contracts/token/ERC20/IERC20.sol';
+import {SafeERC20} from 'openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {GhoEthereum} from 'aave-address-book/GhoEthereum.sol';
+import {MiscEthereum} from 'aave-address-book/MiscEthereum.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {CollectorUtils, ICollector} from 'aave-helpers/src/CollectorUtils.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/src/interfaces/IProposalGenericExecutor.sol';
+
+import {IGhoBucketSteward} from 'src/interfaces/IGhoBucketSteward.sol';
+import {IGhoToken} from 'src/interfaces/IGhoToken.sol';
+import {IGsm} from 'src/interfaces/IGsm.sol';
+import {IGsmRegistry} from 'src/interfaces/IGsmRegistry.sol';
+import {IAaveCLRobotOperator} from 'src/interfaces/IAaveCLRobotOperator.sol';
+import {IOwnableFacilitator} from 'src/interfaces/IOwnableFacilitator.sol';
+import {IGhoReserve} from 'src/interfaces/IGhoReserve.sol';
+
+/**
+ * @title GSM Migration
+ * @author @TokenLogic
+ * - Snapshot: https://snapshot.box/#/s:aavedao.eth/proposal/0xeb3572580924976867073ad9c8012cb9e52093c76dafebd7d3aebf318f2576fb
+ * - Discussion: https://governance.aave.com/t/arfc-launch-gho-on-plasma-set-aci-as-emissions-manager-for-rewards/22994/8
+ */
+contract AaveV3Ethereum_GSMMigration_20251113 is IProposalGenericExecutor {
+  using SafeERC20 for IERC20;
+  using CollectorUtils for ICollector;
+
+  // OwnableFacilitator Constants
+  address public constant OWNABLE_FACILITATOR = 0x616AEe98F73C79FE59548Cfe7631c0baDBdA3165;
+  string public constant OWNABLE_FACILITATOR_NAME = 'OwnableFacilitator Gho GSMs Mainnet';
```

Comment:
```suggestion
  string public constant OWNABLE_FACILITATOR_NAME = 'EthereumGhoReserveGSM';
```
This is probably enough. We can also use `EthereumGSM`.
For reference, other facilitator names: `CoreGhoDirectMinter`, `LidoGhoDirectMinter`, `HorizonGhoDirectMinter`